components/mvp_table/commons/mvp_table.lua

```diff
@@ -0,0 +1,234 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MvpTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Flags = require('Module:Flags')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Team = require('Module:Team')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local _PAGENAME = mw.title.getCurrentTitle().text
+
+local MvpTable = {}
+
+function MvpTable.run(args)
+	args = MvpTable._parseArgs(args)
+	local conditions = MvpTable._buildConditions(args)
+	local queryData = mw.ext.LiquipediaDB.lpdb('match2', {
+		conditions = conditions,
+		query = 'extradata, match2opponents',
+		limit = 5000,
+	})
+
+	if type(queryData) ~= 'table' or not queryData[1] then
+		return ''
+	end
+
+	local mvpList = MvpTable.processData(queryData)
+
+	local output = mw.html.create('table')
+		:addClass('wikitable prizepooltable collapsed')
+		:css('text-align', 'center')
+		:css('margin-top', args.margin .. 'px')
+		:attr('data-opentext', 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList)
+		:attr('data-closetext', 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList)
+		:attr('data-cutafter', args.cutafter + (String.isNotEmpty(args.title) and 1 or 0))
+		:attr('data-definedcutafter', '')
+		:node(MvpTable._header1(args))
+		:node(MvpTable._header2(args))
+
+	for _, item in ipairs(mvpList) do
+		output:node(MvpTable._row(item, args))
+	end
+
+	return output
+end
+
+function MvpTable._parseArgs(args)
+	local parsedArgs = {
+		cutafter = tonumber(args.cutafter) or 5,
+		margin = Logic.readBool(args.margin or 'true') and 20 or 0,
+		points = Logic.readBool(args.points),
+		title = args.title,
+
+		matchGroupIds = {},
+		tournaments = {},
+	}
+
+	for _, matchGroupId in Table.iter.pairsByPrefix(args, 'id') do
+		if String.isNotEmpty(matchGroupId) then
+			table.insert(parsedArgs.matchGroupIds, matchGroupId)
+		end
+	end
+
+	if String.isNotEmpty(args.tournament) then
+		local tournament = args.tournament:gsub(' ', '_')
+		table.insert(parsedArgs.tournaments, tournament)
+	elseif Table.isEmpty(parsedArgs.matchGroupIds) then
+		table.insert(parsedArgs.tournaments, _PAGENAME)
+	end
+	local tournamentIndex = 2
+	while String.isNotEmpty(args['tournament' .. tournamentIndex]) do
```

Comment:
`pairsByPrefix`

Reply:
starts at 2 not 1, so needs more adjusting^^